plugin.rb

```diff
@@ -51,15 +51,15 @@ def review_post(post)
                     links.push(attachment)
                     node = post_html.create_element 'p'# create paragraph element
                     node.inner_html = SiteSetting.file_attachment_whispers_message
-                    attachment.replace '[color=red]' + node + '[/color]' # replace found link with paragraph
+                    attachment.replace '<font color="red">' + node + '</font>'
                     post.cooked = post_html
                     post.save!
                     hasUpdated = true
                 end
             end
 
             if hasUpdated 
-                updated_raw = ActionView::Base.full_sanitizer.sanitize(post.cooked)
+                updated_raw = post.cooked
```

Comment:
Why is this assignment being done, can you just use post.cooked directly?

Reply:
Actually that's a good point

Reply:
I'm not sure about the entire hasUpdated block of logic existing. It should probably just be a continuation of the last if statement